src/odemis/acq/fastem.py

```diff
@@ -632,7 +632,7 @@ def acquire_roa(self, dataflow):
         # The first field is acquired twice, so the timeout must be at least twice the total field time.
         # Use 5 times the total field time to have a wide margin.
         timeout = 5 * total_field_time + 2
-
+        bs_fail = False
```

Comment:
"bs" is not a great abbreviation to use. Let's explicitly write "beam_shift" or "beamshift".
I'd even call it "beam_shift_fail*ed*" to clarify a little further the meaning. Or you could even invert it and call it "force_beam_shift_update".